spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java

```diff
@@ -396,6 +405,26 @@ private void pullCoverageFile(IDevice device) {
     adbPullFile(device, remotePath, coverageFile.getAbsolutePath());
   }
 
+  /**
+   * Pulls coverage file from external storage of device and saves it in local with test description
+   *
+   * @param device
+   * @param testIdentifier
+   */
+  private void pullCoverageFile(IDevice device, String testIdentifier) {
```

Comment:
this feels like it should share implementation with the `pullCoverageFile` method above.  Something like:
```
private void pullCoverageFile(IDevice device) {
  doPullCoverageFile(device, COVERAGE_FILE);
}
private void pullCoverageFile(IDevice device, String testIdentifier) {
  doPullCoverageFile(device, testIdentifier + "_" + COVERAGE_FILE);
}
private void doPullCoverageFile(IDevice device, String localFileName) {
    coverageDir.mkdirs();
    File coverageFile = new File(coverageDir, localFileName;
    logInfo("Pulling Code Coverage file for %s to %s", testIdentifier,
            coverageFile.getAbsolutePath());
    String remotePath;
    try {
      remotePath = getExternalStoragePath(device, COVERAGE_FILE);
    } catch (Exception exception) {
      throw new RuntimeException("error while calculating coverage file path.", exception);
    }
    adbPullFile(device, remotePath, coverageFile.getAbsolutePath());
)

```

Reply:
Thanks Eric for pointing this out. I have fixed this.

Also, the Travis CI job seems to be failing with an error:
`Failed to install the following Android SDK packages as some licences have not been accepted.`
Should I look into this?